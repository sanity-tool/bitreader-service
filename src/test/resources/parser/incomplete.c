typedef const struct Struct * ConstStructRef;

static void acceptConstStructRef(ConstStructRef str) {
}
