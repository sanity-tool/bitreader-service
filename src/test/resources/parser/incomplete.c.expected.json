{
  "functions" : [ {
    "blocks" : [ {
      "instructions" : [ {
        "kind" : "LLVMAlloca",
        "operands" : [ {
          "kind" : "LLVMConstantIntValueKind"
        } ]
      }, {
        "kind" : "LLVMStore",
        "operands" : [ {
          "kind" : "LLVMArgumentValueKind"
        }, {
          "kind" : "LLVMInstructionValueKind"
        } ]
      }, {
        "kind" : "LLVMCall",
        "operands" : [ {
          "kind" : "LLVMMetadataAsValueValueKind"
        }, {
          "kind" : "LLVMMetadataAsValueValueKind"
        }, {
          "kind" : "LLVMFunctionValueKind"
        } ]
      }, {
        "kind" : "LLVMRet",
        "operands" : [ ]
      } ]
    } ],
    "entryBlockIndex" : 0
  } ]
}